### PR TITLE
FiRECERT-2068: Port HdmiInput test cases to FCS

### PR DESCRIPTION
**Pull request by PreethiMaai**

### CODE REVIEW

File: cypress/fixtures/objects/moduleReqId/moduleReqId.json [hunk 1]
```diff
@@ -2583,6 +2583,98 @@
...
+                    {
+                        "method": {
+                            "id": "HdmiInput 1.5",
+                            "description": "Should not be null"
+                        }
+                    }
+                ]
+            },
+            "HdmiInput.onSignalChanged - Positive Scenario: Validate Hdmi1 - signal unstable": {
+                "req": [
```
Comment (neeradanelxsi) [inline cypress/fixtures/objects/moduleReqId/moduleReqId.json:2639]: any test case which does not have a spec document should not have test case added in modulerequirement id

File: cypress/TestCases/FireboltCertification/Manage/HdmiInput.feature [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+         | Scenario                        | Key                                  |
+         | Validate HDMIInput port - Hdmi1 | get hdmiinput port with portId HDMI1 |
+         | Validate HDMIInput port - Hdmi2 | get hdmiinput port with portId HDMI2 |
+
+   @HdmiInput @manageSDK @sdk @transport
+   Scenario Outline: HDMIInput.port - Positive Scenario: <Scenario>
+      When 1st party app invokes the 'Firebolt' API to '<Key>'
+      When 1st party app invokes the 'Firebolt' API to 'get hdmiinput port with portId HDMI1'
+      Then 'Firebolt' platform responds to '1st party app' with '<Validation_Key>'
+
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/Manage/HdmiInput.feature:27]: setter validation also to be added after we call setter

Comment (PreethiMaai) [reply cypress/TestCases/FireboltCertification/Manage/HdmiInput.feature:28]: We don't have setter validation before, Now added setter validation similar to 2044. Pulled 2044 and tested.

**Review by neeradanelxsi — APPROVED** [review]

**Review by kschrief — APPROVED** [review]

File: cypress/fixtures/modules/secondscreen.json [hunk 3]
```diff
@@ -18,5 +18,20 @@
...
         "version": "1.7"
+    },
+    "ONLAUNCHREQUEST_EVENT": {
+        "method": "secondscreen.onLaunchRequest",
+        "result":{
+            "type": "dial",
+            "version": "1.7",
+            "data": "{\"code\":\"AQDPQZiQcb3KQ7gY7yy5tHTMbbkGHR9Zjp-KL53H3eKBZIeAt7O9UKYPu6B21l2UZVmIqkFXDXBmXvK4g2e3EgZtjMNmKPsTltgnRl95DImtOXjSpWtTjSaOkW4w1kZKUTwLKdwVWTzBVH8ERHorvLU6vCGOVHxXt65LNwdl5HKRweShVC1V9QsyvRnQS61ov0UclmrH_xZML2Bt-Q-rZFjey5MjwupIb4x4f53XUJMhjHpDHoIUKrjpdPDQvK2a\",\"friendlyName\":\"Operator_TX061AEI\",\"UDN\":\"608fef11-2800-482a-962b-23a6690c93c1\"}"
+        }
+    },
```
Comment (neeradanelxsi) [inline cypress/fixtures/modules/secondscreen.json:29]: can we give some dummy value here

Comment (PreethiMaai) [reply cypress/fixtures/modules/secondscreen.json:29]: yes updated

**Review by Abhishk123 — APPROVED** [review]

**Pull request merged**